docs/content/en/docs/troubleshooting/troubleshooting.md

```diff
@@ -439,6 +439,27 @@ Status:
   Phase:                   Running
 ```
 
+### Machine gets stuck at `Provisioned` state without `providerID` set
+
+The VM can be created in the provider infrastructure with proper IP assigned but running `kubectl get machines -n eksa-system` indicates that the machine is in `Provisioned` state and never gets to `Running`.
+
+Check the CAPI controller manager log with `kubectl logs -f capi-controller-manager-7f754cf76b-g92ht -n capi-system`:
```

Comment:
how do we usually show things like this where the deployment name will be somewhat random? do we just assume they know to find their specific one?

Reply:
updated to use label filter